src/sidebar/templates/login-control.html

```diff
@@ -35,7 +31,7 @@
     <li class="dropdown-menu__row">
       <a class="dropdown-menu__link" ng-click="vm.onShowHelpPanel()">Help</a>
     </li>
-    <li class="dropdown-menu__row" ng-if="vm.auth.status === 'logged-in' && !vm.isThirdPartyUser()">
+    <li class="dropdown-menu__row" ng-if="vm.auth.status === 'logged-in'">
```

Comment:
Now that I think about it we should still hide the log out button on publisher group sites if no `onLogoutRequested` is provided. Will fix this